plugins/palo_alto_pan_os/setup.py

```diff
@@ -3,7 +3,7 @@
 
 
 setup(name="palo_alto_pan_os-rapid7-plugin",
-      version="6.1.3",
+      version="6.1.4",
```

Comment:
This should be 6.1.3

Reply:
We leave version, because we added information about connection to description of every action. If you would like we can of course reuse previous version.